build_tools/github_actions/configure_ci.py

```diff
@@ -254,6 +263,11 @@ def get_benchmark_presets(is_pr: bool, trailers: Mapping[str, str],
   """
   if not is_pr:
     preset_options = ["all"]
+  elif (LLVM_INTEGRATION_PR_TITLE_PATTERN.search(os.environ["PR_TITLE"]) or
+        LLVM_INTEGRATION_BRANCH_PATTERN.search(os.environ["PR_BRANCH"]) or
+        LLVM_INTEGRATE_LABEL in labels):
```

Comment:
How about extracting this logic up a level, to match how `is_pr` is handled?

```python
def main():
  is_pr = os.environ["GITHUB_EVENT_NAME"] == "pull_request"
  is_llvm_integrate = (
      LLVM_INTEGRATION_PR_TITLE_PATTERN.search(os.environ["PR_TITLE"]) or
      LLVM_INTEGRATION_BRANCH_PATTERN.search(os.environ["PR_BRANCH"]) or
      LLVM_INTEGRATE_LABEL in labels)
...

  output = {
      "should-run": json.dumps(should_run_ci(is_pr, is_llvm_integrate, trailers)),
```

```suggestion
  if not is_pr:
    preset_options = ["all"]
  elif is_llvm_integrate:
```

Reply:
It still needs to access the variable `labels` returned from `get_trailers_and_labels` and the check is only available for the pull request. But this still give a wider access to `is_llvm_integate`. Moved the code.